rotary_utils/repair.py

```diff
@@ -472,59 +499,26 @@ def iterate_linking_contig_ends(contig_record: SeqIO.SeqRecord, bam_filepath: st
             assembly_attempts = assembly_attempts + 1
             continue
```

Comment:
@jmtsuji I would leave a comment after the `continue` stating why you are skipping this loop iteration. Apply to all continues. So basically, every time you run into some sort of assembly failure you increment the number of attempts and the break at the top checks the number of attempts and breaks if they are to high?

Reply:
Yes, your understanding of the logic is correct. I added more comments and tried to make the syntax more consistent between each case where assembly/stitching fails, to make the logic clearer.